### ISSUE TITLE
Form field border color contrast

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1902]
> Border of all form fields within iframes fail color contrast requirements. Borders test at 1.6:1 and the minimum requirement is 3:1. 
> Recommend using a darker color such as #949494.

## Comments

Agree.
